src/app/portlets/dot-edit-page/content/shared/iframe-edit-mode.css.ts

```diff
@@ -11,6 +11,7 @@ export const EDIT_PAGE_CSS = `
         border: solid 1px #53c2f9;
         min-height: 120px;
         margin-bottom:40px;
+        padding: 0 16px 16px;
```

Comment:
Do we need this changes now that we're not calc anything?

Reply:
not really, but IMO makes it more clean

do you want me to revert it?